dotnet/src/SemanticKernel.Skills/Skills.Memory.Qdrant/QdrantVectorDbClient.cs

```diff
@@ -0,0 +1,512 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel.Memory.Storage;
+using Microsoft.SemanticKernel.Skills.Memory.Qdrant.Diagnostics;
+using Microsoft.SemanticKernel.Skills.Memory.Qdrant.Http;
+using Microsoft.SemanticKernel.Skills.Memory.Qdrant.Http.ApiSchema;
```

Comment:
this client should be equivalent to a client written by a third party, without any dependency on the kernel. All the mapping to what the kernel does with this should be in QdrantMemoryStore

Reply:
@dluc will you be willing to accept this PR as is knowing that the next work I have in the pipeline is to merge the memory interfaces, make MemoryRecord the IMemoryStore return type, and eliminate generics from IMemoryStore? I can address the issue in this PR, but it's a pretty annoying, wide change that will be overwritten in the design we discussed earlier today.

Reply:
@dluc If what @awharrison-28 noted is okay I can resolve this conversation.